definitions/consented/json/CaseEventToFields/CaseEventToFields.json

```diff
@@ -1284,7 +1284,6 @@
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
     "ShowSummaryChangeOption": "N",
-    "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/case-orchestration/consented/set-defaults",
```

Comment:
this call back sets the organisation policy. Are you setting anywhere else.

Reply:
you don't need to set the organisation policy here. This is amend case

Reply:
This is only case - where its set is FormA. In case on Consent Paper case - This will not be set. In that case NOC flow will break.

Reply:
also this jounrey used by finrem exception records as well. @gmikerlemos

Reply:
I added back the set defaults cal